pom.xml

```diff
@@ -659,10 +766,70 @@
             <jdk.tls.client.protocols>TLSv1.2</jdk.tls.client.protocols>
             <bucketeer.slack.error_channel_id>dev-null</bucketeer.slack.error_channel_id>
           </systemPropertyVariables>
+          <skipTests>${skipUTs}</skipTests>
           <argLine>${jacoco.agent.arg}</argLine>
         </configuration>
       </plugin>
 
+      <!-- This incantation of Failsafe runs the non-Kakadu integration and feature-flag tests -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>**/*Test.java</exclude>
+            <exclude>**/*KakaduIT.java</exclude>
+          </excludes>
+          <systemPropertyVariables>
+            <vertx.logger-delegate-factory-class-name>io.vertx.core.logging.SLF4JLogDelegateFactory</vertx.logger-delegate-factory-class-name>
+            <vertx-config-path>${project.basedir}/target/test-classes/test-config.properties</vertx-config-path>
+            <jdk.tls.client.protocols>TLSv1.2</jdk.tls.client.protocols>
+            <jdwp.client.config>${jdwp.client.config}</jdwp.client.config>
+            <jdwp.host.port>${jdwp.host.port}</jdwp.host.port>
```

Comment:
I'm not able to figure out how to attach a debugger to these containers, btw. Like you said this is going to require a task ticket

Reply:
Do you think it's best to remove this config info until that time, so as not to give the impression that we already know how to do this?

Reply:
yeah I think so

Reply:
Do you want to do this with your README change, or I can... either way.